recipes-mender/mender/mender_0.1.bb

```diff
@@ -32,6 +32,7 @@ SYSTEMD_SERVICE_${PN} = "mender.service"
 FILES_${PN} += "${systemd_unitdir}/system/mender.service \
                 ${sysconfdir}/mender.conf \
                "
+SYSTEMD_AUTO_ENABLE = "disable"
```

Comment:
Yeah, ideally they should just disable it in bbappend, but I'm completely lost at this as to what we actually want. There are conflicting opinions about what's good for integration testing and customers doing evaluation testing. I have no preference here.